mono/metadata/exception-internals.h

```diff
@@ -41,4 +41,16 @@ typedef int (*MonoGetSeqPointFunc) (MonoDomain *domain, MonoMethod *method, gint
 void
 mono_install_get_seq_point (MonoGetSeqPointFunc func);
 
+MonoExceptionHandle
```

Comment:
I kind think these really should be <old name>_handle for clarity, even if <old name> is not ideal.

Reply:
Why did you name it `mono_new_...` then?

Reply:
It was suggested by Rodrigo and agreed by you.

Reply:
Maybe agreed by you in a different context - the object allocation functions.

Reply:
As discussed yesterday, we should have `mono_exception_new_by_name` or `mono_exception_new_handle_by_name`, but I would prefer `mono_exception_new_by_name`

Reply:
These functions can be static so far. How about mono_new_exception_thread_abort, mono_new_exception_thread_interrupted? Also mono_exception_new_thread...?

Reply:
It is not clear if exception is a type or a subsystem/namespace, and if one should say type_new or new_type, or subsystem_new; new_subsystem obviously seems wrong.
If one wants to leave type vs. subsystem in the eye of the beholder, then foo_new does read ok either way. Anyway I made it mono_exception_new_xxx for all of them, static and non-static, though the static ones could have shorter names, though naming them as if non-static helps if they have to become non-static (extern).